tests/test_fuzzy.py

```diff
@@ -34,7 +34,9 @@ def test_unbiased(self):
 
     def test_mock(self):
         options = [1, 2, 3]
-        fake_choice = lambda d: sum(d)
+
+        def fake_choice(d):
+            return sum(d)
```

Comment:
I disagree with this kind of change, which reduces readability.

Can you point to the flake8 flag to adjust to remove that warning?